implementations/.prettierrc

```diff
@@ -3,11 +3,11 @@
     {
```

Comment:
I think this prettier config should not be edited.

It was intended to match with the one defined in **[lukso-network/lsp-smart-contracts](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/.prettierrc)**

Reply:
no problem, will change this config back and run the formatter as the first commit in this PR.